src/pymovements/gaze/gaze_dataframe.py

```diff
@@ -402,11 +405,51 @@ def unnest(
             Name of input columns to be unnested into several component columns.
         output_columns:
             Name of the resulting tuple columns.
+        output_suffixes:
+            Suffixes to append to the column names.
+
+        Raises
+        ------
+        ValueError
+            If both output_columns and output_suffixes are specified.
+            If number of output columns / suffixes does not match number of components.
+            If output columns / suffixes are not unique.
+        AttributeError
+            If number of components is not 2, 4 or 6.
         """
+
+        checks.check_is_mutual_exclusive(
+            output_columns=output_columns,
+            output_suffixes=output_suffixes,
+        )
+        _check_n_components(self.n_components)
+
+        # make mypy happy
+        col_names = output_columns if output_columns is not None else []
+
+        if output_columns is None and output_suffixes is None:
+            if self.n_components == 2:
+                output_suffixes = ['_x', '_y']
+            elif self.n_components == 4:
+                output_suffixes = ['_xl', '_yl', '_xr', '_yr']
+            elif self.n_components == 6:
```

Comment:
there needs to be an else such that coverage is 100%.

you can remove the calling of `_check_n_components(self.n_components)` in the lines before and just put this here:

```python
else:
    raise AttributeError(f'n_components must be either 2, 4 or 6 but is {self.n_components}')
```